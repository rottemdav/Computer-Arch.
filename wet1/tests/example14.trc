4 3 24 0 local_history local_tables not_using_share
0x6273b440 T 0xba108
0x71c6e04 T 0x981adc
0x4a9fd0f4 T 0x2aa00318
0xafe50954 T 0x9f9e28
0x666dc N 0x74890
0x8b584298 N 0xe890cc
0x88aab478 T 0x7fc
0x0543e404 T 0xe4158e4c
0x8d10 N 0x4af47f8
0x0086c T 0x9ebe074
0x6273b440 N 0xba108
0xc4ee2f4 T 0x23c
0xd544 T 0xa5367e84
0xe24a704 T 0x1b08
0x58ae4 N 0x6582bc
0x8d10 N 0x4af47f8
0x71c6e04 N 0x981adc
0x46b94 T 0x8b4
0x91a708 T 0x48d70
0xbe48 N 0x694
0xe24a704 N 0x1b08
0x7aa4 N 0x41f31a4c
0x88aab478 T 0x7fc
0xe24a704 T 0x1b08
0x6273b440 N 0xba108
0xafe50954 T 0x9f9e28
0x7aa4 N 0x41f31a4c
0x6273b440 T 0xba108
0x0543e404 N 0xe4158e4c
0x4a9fd0f4 N 0x2aa00318
0x1fc T 0x48f6db4
0x76040 N 0x000330a0
0x562c N 0x927a60
0x0a8 N 0x066bec4
0xd31d8 N 0xb35f6d58
0xbe48 N 0x694
0x186990ac T 0x7fa28
0x8b584298 N 0xe890cc
0x047b4 N 0x32c
0x76040 N 0x000330a0
0xdfcfc4 T 0x30c
0x0543e404 N 0xe4158e4c
0xafe50954 N 0x9f9e28
0x1fc T 0x48f6db4
0x7aa4 N 0x41f31a4c
0xd31d8 N 0xb35f6d58
0xe24a704 N 0x1b08
0x76040 N 0x000330a0
0x562c T 0x927a60
0x7d255448 T 0x77d1b28
0x562c N 0x927a60
0x5ad69b2c T 0x558c
0x71c6e04 N 0x981adc
0x6273b440 T 0xba108
0x46b94 T 0x8b4
0x0a8 N 0x066bec4
0xafe50954 T 0x9f9e28
0xd544 T 0xa5367e84
0xbe48 T 0x694
0x186990ac T 0x7fa28
0x6273b440 T 0xba108
0x666dc T 0x74890
0x047b4 T 0x32c
0x6273b440 T 0xba108
0x7d255448 T 0x77d1b28
0x88aab478 T 0x7fc
0x91a708 T 0x48d70
0x666dc N 0x74890
0x666dc T 0x74890
0x920f98 N 0x35da8710
0xdfcfc4 T 0x30c
0xd544 N 0xa5367e84
0x8b584298 N 0xe890cc
0x8d10 T 0x4af47f8