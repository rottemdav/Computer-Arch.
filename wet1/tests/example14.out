0x6273b440 N 0x6273b444
0x71c6e04 N 0x71c6e08
0x4a9fd0f4 N 0x4a9fd0f8
0xafe50954 N 0xafe50958
0x666dc N 0x666e0
0x8b584298 N 0x8b58429c
0x88aab478 N 0x88aab47c
0x543e404 N 0x543e408
0x8d10 N 0x8d14
0x86c N 0x870
0x6273b440 N 0x6273b444
0xc4ee2f4 N 0xc4ee2f8
0xd544 N 0xd548
0xe24a704 N 0xe24a708
0x58ae4 N 0x58ae8
0x8d10 N 0x8d14
0x71c6e04 N 0x71c6e08
0x46b94 N 0x46b98
0x91a708 N 0x91a70c
0xbe48 N 0xbe4c
0xe24a704 N 0xe24a708
0x7aa4 N 0x7aa8
0x88aab478 N 0x88aab47c
0xe24a704 N 0xe24a708
0x6273b440 N 0x6273b444
0xafe50954 N 0xafe50958
0x7aa4 N 0x7aa8
0x6273b440 N 0x6273b444
0x543e404 N 0x543e408
0x4a9fd0f4 N 0x4a9fd0f8
0x1fc N 0x200
0x76040 N 0x76044
0x562c N 0x5630
0xa8 N 0xac
0xd31d8 N 0xd31dc
0xbe48 N 0xbe4c
0x186990ac N 0x186990b0
0x8b584298 N 0x8b58429c
0x47b4 N 0x47b8
0x76040 N 0x76044
0xdfcfc4 N 0xdfcfc8
0x543e404 N 0x543e408
0xafe50954 N 0xafe50958
0x1fc N 0x200
0x7aa4 N 0x7aa8
0xd31d8 N 0xd31dc
0xe24a704 N 0xe24a708
0x76040 N 0x76044
0x562c N 0x5630
0x7d255448 N 0x7d25544c
0x562c N 0x5630
0x5ad69b2c N 0x5ad69b30
0x71c6e04 N 0x71c6e08
0x6273b440 N 0x6273b444
0x46b94 N 0x46b98
0xa8 N 0xac
0xafe50954 N 0xafe50958
0xd544 N 0xd548
0xbe48 N 0xbe4c
0x186990ac N 0x186990b0
0x6273b440 N 0x6273b444
0x666dc N 0x666e0
0x47b4 N 0x47b8
0x6273b440 N 0x6273b444
0x7d255448 N 0x7d25544c
0x88aab478 N 0x88aab47c
0x91a708 N 0x91a70c
0x666dc N 0x666e0
0x666dc N 0x666e0
0x920f98 N 0x920f9c
0xdfcfc4 N 0xdfcfc8
0xd544 N 0xd548
0x8b584298 N 0x8b58429c
0x8d10 N 0x8d14
flush_num: 39, br_num: 74, size: 296b
